README.md

```diff
@@ -5,17 +5,13 @@ additional functionalities that allow applications to communicate with a remote
 Spark cluster using the Spark Connect protocol without requiring additional steps.
 
 ## Install
-
-.. code-block:: console
```

Comment:
I think this syntax might be used for some test automation. Do you have specific reason to change it?